app/metrics.yaml

```diff
@@ -4260,3 +4260,44 @@ tabs:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-08-01"
+
+banner_open_in_app:
+  displayed:
+    type: event
+    description: |
+      Open in App banner was shown.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/16828
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/17049
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-06-01"
```

Comment:
Can we match the expiration dates on the other telemetry events (8/1 instead of 6/1)? This makes it easier for renewal